lib/loadMtl.js

```diff
@@ -387,6 +387,10 @@ function createDiffuseAlphaTexture(diffuseTexture, alphaTexture, options) {
         return diffuseTexture;
     }
 
+    if (diffuseTexture.path === alphaTexture.path) {
+        return diffuseTexture;
```

Comment:
I think `diffuseTexture.transparent = true` needs to be set here in order for the blend mode to be set correctly. And add these lines to the new test to ensure that:

```
expect(material.alphaMode).toBe('BLEND');
expect(material.doubleSided).toBe(true);
```